src/python/WMCore/DataStructs/MathStructs/ContinuousSummaryHistogram.py

```diff
@@ -8,6 +8,8 @@
 
 @author: dballest
 """
+from __future__ import division
+from past.utils import old_div
```

Comment:
How about we use only future division here?

Reply:
I left old_div because I was not sure how to change it. I tried to use python3 division, let's see if jenkins agrees with my intuition

Reply:
It seems that jenkins liked the changes!